subsys/nrf_security/src/drivers/cracen/sxsymcrypt/src/cmdma.c

```diff
@@ -52,14 +54,16 @@ void sx_cmdma_start(struct sx_dmactl *dma, size_t privsz, struct sxdesc *indescs
 #ifdef CONFIG_DCACHE
 	m = (struct sxdesc *)(dma->mapped + sizeof(struct sx_dmaslot));
 	for (; m != DMA_LAST_DESCRIPTOR; m = m->next) {
+		sys_cache_data_flush_and_invd_range(&m, sizeof(*m));
 		sys_cache_data_flush_and_invd_range(m->addr, m->sz & DMA_SZ_MASK);
 	}
 	m = (struct sxdesc *)(dma->mapped + offsetof(struct sx_dmaslot, outdescs));
 	for (; m != DMA_LAST_DESCRIPTOR; m = m->next) {
+		sys_cache_data_flush_and_invd_range(&m, sizeof(*m));
```

Comment:
This (and the one above) is incorrect, you want to have either
```suggestion
		sys_cache_data_flush_and_invd_range(&m, sizeof(m));
```
or
```suggestion
		sys_cache_data_flush_and_invd_range(m, sizeof(*m));
```

Reply:
yes totally, thanks